lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridgeConfigurationOptions.cs

```diff
@@ -153,7 +154,7 @@ public class TraceListenersSet : Name<ISet<TraceListener>>
         {
         }
 
-        [NamedParameter("Custom Trace Level", "TraceLevel", defaultValue: _verboseLevel)]
+        [NamedParameter("Custom Trace Level", "TraceLevel", defaultValue: _infoLevel)]
```

Comment:
Given that our tests rely on logs, doesn't this represent a breaking change? We now log less by default, right?

Reply:
This is the log level for driver, not just for test. For prod code, we usually set default at Info level rather than Verbose level. 
Client can set it as part of the driver configuration. Especially in test driver configuration, they can always turn it into Verbose.

Reply:
Created REEF-1897 for it and send PL.
https://github.com/apache/reef/pull/1386